packages/backend-function/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@aws-amplify/backend-output-storage": "0.2.6",
     "@aws-amplify/plugin-types": "^0.6.0",
+    "@aws-sdk/client-ssm": "^3.398.0",
```

Comment:
our lint rules might yell at you, but technically this doesn't need to be a dependency here. It's only used in the lambda code which comes with the sdk pre-installed

Reply:
Yup eslint is yelling at me when this is removed and I also mock out SSM's `getParameters` for testing, I moved this to `devDependencies` instead.